indexd/auth/drivers/alchemy.py

```diff
@@ -131,5 +131,11 @@ def authz(self, method, resource):
             # operations on the record. For now, admin = access to `/programs`.
             # TODO: Figure out how to handle Gen3 operational admins in a better way
             resource = ["/programs"]
-        if not self.arborist.auth_request(get_jwt_token(), "indexd", method, resource):
-            raise AuthError("Permission denied.")
+
+        try:
+            if not self.arborist.auth_request(
```

Comment:
In terms of unit testing I would try to mock `self.arborist.auth_request` to return the 401 error. Pytest is really cool like that. Add a test that calls this `authz` function or whatever the endpoint calls the `authz` function and mock the return for `auth_request`